server/data/manageUsersApiClient.ts

```diff
@@ -22,4 +22,8 @@ export default class ManageUsersApiClient {
     logger.info('Getting user details: calling HMPPS Manage Users Api')
     return ManageUsersApiClient.restClient(token).get<User>({ path: '/users/me' })
   }
+
+  getNamedUser(token: string, username: string): Promise<User> {
```

Comment:
I wouldn't mind if this method was simply called `getUser()` :)

Reply:
I named it `getNamedUser` as there is already a function in that class that is called `getUser` which gets the current user's profile using their token.

Reply:
This is now resolved in the moving of the function.